src/tocsym.c

```diff
@@ -285,7 +290,10 @@ Symbol *toSymbol(Dsymbol *s)
                 Symbol *s = symbol_calloc(id);
                 slist_add(s);
 
-                s->prettyIdent = fd->toPrettyChars(true);
+                if (global.params.isWindows)
+                    s->prettyIdent = fd->toPrettyChars(true);
+                else
+                    s->prettyIdent = fd->toChars();
                 s->Sclass = SCglobal;
```

Comment:
So you want to parse the human readable function type, e.g. `void function() foo(int arg)`? Sounds like it would require half of a D parser.